src/components/app/content/cards/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import "./index.css"
+import Card from "./card";
+
+function Cards(props) {
+    return (
+        <div className="cards">
+            {props.cards.map(card => <Card readonly={props.readOnly} initHeader={card.header} initBody={card.body}
+                                           key={card.id}/>)}
```

Comment:
I would advice to pass the card data as a single object and then destruct it in Card component as required. In that way you won't need to rewrite the logic once number of card data properties increase.